resources/charts/kubelet/templates/kubelet-ds.yaml

```diff
@@ -0,0 +1,137 @@
+        # this kubelet as well.
+        args:
+        - |
+          echo "./hyperkube kubelet \
+          --node-ip=$(HOST_IP)  \
+          --anonymous-auth=false \
+          --authentication-token-webhook \
+          --authorization-mode=Webhook \
+          --client-ca-file=/etc/kubernetes/ca.crt \
+          --cluster_dns={{ .Values.clusterDNS }} \
+          --cluster_domain={{ .Values.clusterDomain }} \
+          --cni-conf-dir=/etc/kubernetes/cni/net.d \
+          --config=/etc/kubernetes/kubelet.config \
+          --kubeconfig=/etc/kubernetes/kubeconfig \
+          --lock-file=/var/run/lock/kubelet.lock \
+          --network-plugin=cni \
+          --pod-manifest-path=/etc/kubernetes/manifests \
+          --read-only-port=0 \
+          --volume-plugin-dir=/var/lib/kubelet/volumeplugins \
+          --node-labels=$(grep NODE_LABELS /etc/kubernetes/kubelet.env | cut -d'"' -f2) \
+          --register-with-taints=$(grep NODE_TAINTS /etc/kubernetes/kubelet.env | cut -d'"' -f2)
+          " | tee /tmp/start-kubelet.sh &&
+          chmod +x /tmp/start-kubelet.sh &&
+          /tmp/start-kubelet.sh
+        env:
+        - name: HOST_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /var/lib/cni
+          name: coreos-var-lib-cni
+          readOnly: false
+        - mountPath: /var/lib/calico
+          name: coreos-var-lib-calico
+          readOnly: true
+        - mountPath: /opt/cni/bin
+          name: coreos-opt-cni-bin
+          readOnly: true
+        # TODO check if this is needed
+        - name: dev
+          mountPath: /dev
+        - name: run
+          mountPath: /run
+        - name: sys
+          mountPath: /sys
+          readOnly: true
+        - name: etc-kubernetes
+          mountPath: /etc/kubernetes
+          readOnly: true
+        - name: var-lib-docker
+          mountPath: /var/lib/docker
+        - name: var-lib-kubelet
+          mountPath: /var/lib/kubelet
+          mountPropagation: Bidirectional
+          # Logs from the kubelet pods
+        - name: logs
+          mountPath: /var/log/pods
+          # This is mounted from host to make sure that the kubelet showcases OS as Flatcar and not
+          # Debian from the hyperkube image.
+        - name: os-release
+          mountPath: /etc/os-release
+          readOnly: true
+      hostNetwork: true
+      hostPID: true
+      # Tolerate all the taints. This ensures that the pod runs on all the nodes.
+      # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+      tolerations:
+      - operator: "Exists"
+      volumes:
+      - name: coreos-var-lib-cni
+        hostPath:
+          path: /var/lib/cni
+      - name: coreos-var-lib-calico
+        hostPath:
+          path: /var/lib/calico
```

Comment:
Why is this needed? From my tests, kubelet should be independent from the CNI you use.

Reply:
After I removed the mount in this commit: https://github.com/kinvolk/terraform-render-bootkube/pull/36/commits/236329fd41a33b431bec64eaf7e379a0f608aa10

```yaml
      - name: coreos-var-lib-calico
        hostPath:
          path: /var/lib/calico
```

And installation failed as:

```bash
...
module.packet-lithium.null_resource.bootkube-start: Still creating... [21m30s elapsed]
module.packet-lithium.null_resource.bootkube-start: Still creating... [21m40s elapsed]
module.packet-lithium.null_resource.bootkube-start: Still creating... [21m50s elapsed]
module.packet-lithium.null_resource.bootkube-start (remote-exec): Job for bootkube.service failed because the control process exited with error code.
module.packet-lithium.null_resource.bootkube-start (remote-exec): See "systemctl status bootkube.service" and "journalctl -xe" for details.
                                                                                      
                                                                                      
Error: error executing "/tmp/terraform_1368907443.sh": Process exited with status 1
```

and here are the logs from bootkube

```log
-- Logs begin at Wed 2020-01-29 12:32:13 UTC, end at Wed 2020-01-29 12:57:46 UTC. --
Jan 29 12:33:30 suraj-cluster-controller-0 systemd[1]: Starting Bootstrap a Kubernetes cluster...
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: v1.17.2: Pulling from hyperkube-amd64
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 346aee5ea5bc: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 9c47fde751af: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: be2693a52daf: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 6b69eb11d043: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 0abeb1500767: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 6b69eb11d043: Waiting
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 0abeb1500767: Waiting
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 4062d80041b7: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 23b6daf06fc2: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 7d7512f8b205: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 8d523ca27b75: Pulling fs layer
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 23b6daf06fc2: Waiting
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 7d7512f8b205: Waiting
Jan 29 12:33:32 suraj-cluster-controller-0 bootkube-start[1436]: 8d523ca27b75: Waiting
Jan 29 12:33:33 suraj-cluster-controller-0 bootkube-start[1436]: 9c47fde751af: Verifying Checksum
Jan 29 12:33:33 suraj-cluster-controller-0 bootkube-start[1436]: 9c47fde751af: Download complete
Jan 29 12:33:33 suraj-cluster-controller-0 bootkube-start[1436]: be2693a52daf: Verifying Checksum
Jan 29 12:33:33 suraj-cluster-controller-0 bootkube-start[1436]: be2693a52daf: Download complete
Jan 29 12:33:34 suraj-cluster-controller-0 bootkube-start[1436]: 346aee5ea5bc: Verifying Checksum
Jan 29 12:33:34 suraj-cluster-controller-0 bootkube-start[1436]: 346aee5ea5bc: Download complete
Jan 29 12:33:34 suraj-cluster-controller-0 bootkube-start[1436]: 6b69eb11d043: Verifying Checksum
Jan 29 12:33:34 suraj-cluster-controller-0 bootkube-start[1436]: 6b69eb11d043: Download complete
Jan 29 12:33:34 suraj-cluster-controller-0 bootkube-start[1436]: 0abeb1500767: Verifying Checksum
Jan 29 12:33:34 suraj-cluster-controller-0 bootkube-start[1436]: 0abeb1500767: Download complete
Jan 29 12:33:35 suraj-cluster-controller-0 bootkube-start[1436]: 23b6daf06fc2: Verifying Checksum
Jan 29 12:33:35 suraj-cluster-controller-0 bootkube-start[1436]: 23b6daf06fc2: Download complete
Jan 29 12:33:36 suraj-cluster-controller-0 bootkube-start[1436]: 8d523ca27b75: Verifying Checksum
Jan 29 12:33:36 suraj-cluster-controller-0 bootkube-start[1436]: 8d523ca27b75: Download complete
Jan 29 12:33:36 suraj-cluster-controller-0 bootkube-start[1436]: 346aee5ea5bc: Pull complete
Jan 29 12:33:37 suraj-cluster-controller-0 bootkube-start[1436]: 9c47fde751af: Pull complete
Jan 29 12:33:37 suraj-cluster-controller-0 bootkube-start[1436]: be2693a52daf: Pull complete
Jan 29 12:33:37 suraj-cluster-controller-0 bootkube-start[1436]: 4062d80041b7: Verifying Checksum
Jan 29 12:33:37 suraj-cluster-controller-0 bootkube-start[1436]: 4062d80041b7: Download complete
Jan 29 12:33:37 suraj-cluster-controller-0 bootkube-start[1436]: 6b69eb11d043: Pull complete
Jan 29 12:33:37 suraj-cluster-controller-0 bootkube-start[1436]: 0abeb1500767: Pull complete
Jan 29 12:33:38 suraj-cluster-controller-0 bootkube-start[1436]: 7d7512f8b205: Verifying Checksum
Jan 29 12:33:38 suraj-cluster-controller-0 bootkube-start[1436]: 7d7512f8b205: Download complete
Jan 29 12:33:53 suraj-cluster-controller-0 bootkube-start[1436]: 4062d80041b7: Pull complete
Jan 29 12:33:55 suraj-cluster-controller-0 bootkube-start[1436]: 23b6daf06fc2: Pull complete
Jan 29 12:34:11 suraj-cluster-controller-0 bootkube-start[1436]: 7d7512f8b205: Pull complete
Jan 29 12:34:12 suraj-cluster-controller-0 bootkube-start[1436]: 8d523ca27b75: Pull complete
Jan 29 12:34:12 suraj-cluster-controller-0 bootkube-start[1436]: Digest: sha256:adad8b9f5271e6917d7463d66c2474c9d596ec57bbd6e7f69d61222a395ff261
Jan 29 12:34:12 suraj-cluster-controller-0 bootkube-start[1436]: Status: Downloaded newer image for k8s.gcr.io/hyperkube-amd64:v1.17.2
Jan 29 12:34:14 suraj-cluster-controller-0 bootkube-start[1436]: Downloading ACI:  0 B/24.3 MB
Jan 29 12:34:14 suraj-cluster-controller-0 bootkube-start[1436]: Downloading ACI:  16.4 KB/24.3 MB
Jan 29 12:34:15 suraj-cluster-controller-0 bootkube-start[1436]: Downloading ACI:  17.7 MB/24.3 MB
Jan 29 12:34:15 suraj-cluster-controller-0 bootkube-start[1436]: Downloading ACI:  24.3 MB/24.3 MB
Jan 29 12:34:19 suraj-cluster-controller-0 bootkube-start[1436]: Starting temporary bootstrap control plane...
Jan 29 12:34:19 suraj-cluster-controller-0 bootkube-start[1436]: Waiting for api-server...
Jan 29 12:34:24 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:24.814351    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:29 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:29.789952    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:34 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:34.789992    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:39 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:39.790228    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:44 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:44.790217    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:49 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:49.790041    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:54 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:54.790036    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:34:59 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:34:59.790605    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:35:04 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:35:04.791467    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 0
Jan 29 12:35:15 suraj-cluster-controller-0 bootkube-start[1436]: W0129 12:35:15.527838    1436 util.go:21] Unable to determine api-server readiness: API Server http status: 500
Jan 29 12:35:19 suraj-cluster-controller-0 bootkube-start[1436]: WARNING: /assets/manifests does not exist, not creating any self-hosted assets.
Jan 29 12:35:19 suraj-cluster-controller-0 bootkube-start[1436]: loading chart --- calico
Jan 29 12:35:23 suraj-cluster-controller-0 bootkube-start[1436]: creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)creating 1 resource(s)Clearing discovery cachebeginning wait for 14 resources with timeout of 1m0screating 18 resource(s)Release-created :: calico
Jan 29 12:35:23 suraj-cluster-controller-0 bootkube-start[1436]: loading chart --- kubelet
Jan 29 12:35:23 suraj-cluster-controller-0 bootkube-start[1436]: creating 1 resource(s)Release-created :: kubelet
Jan 29 12:35:23 suraj-cluster-controller-0 bootkube-start[1436]: loading chart --- kubernetes
Jan 29 12:35:24 suraj-cluster-controller-0 bootkube-start[1436]: creating 44 resource(s)Release-created :: kubernetes
Jan 29 12:35:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:35:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:35:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:35:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:35:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:35:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:35:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:35:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:35:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:35:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:35:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:35:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:35:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:35:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:35:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:35:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:36:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:36:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:36:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:36:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:36:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:36:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:36:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:36:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:36:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:36:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:36:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:36:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:36:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:36:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer        DoesNotExist
Jan 29 12:36:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver        DoesNotExist
Jan 29 12:36:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:36:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Pending
Jan 29 12:36:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Pending
Jan 29 12:36:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:36:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:36:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:36:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Pending
Jan 29 12:36:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:36:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:36:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:36:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:36:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:36:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:36:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:37:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:37:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:37:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:37:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:37:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:37:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:37:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:37:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:37:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:37:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:37:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:37:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:37:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:37:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:38:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:38:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:38:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:38:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:38:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:38:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:38:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:38:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:38:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:38:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:38:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:38:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:38:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:38:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:38:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:38:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:38:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:38:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:39:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:39:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:39:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:39:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:39:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:39:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:39:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:39:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:39:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:39:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:39:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:39:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:39:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:39:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:39:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:39:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:39:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:39:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:40:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:40:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:40:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:40:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:40:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:40:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:40:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:40:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:40:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:40:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:40:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:40:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:40:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:40:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:40:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:40:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:40:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:40:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:40:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:41:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:41:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:41:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:41:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:41:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:41:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:41:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:41:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:41:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:41:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:41:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:41:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:41:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:41:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:41:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:41:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:42:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:42:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:42:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:42:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:42:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:42:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:42:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:42:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:42:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:42:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:42:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:42:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:42:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:42:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:42:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:42:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:42:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:42:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:43:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:43:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:43:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:43:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:43:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:43:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:43:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:43:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:43:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:43:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:43:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:43:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:44:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:44:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:44:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:44:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:44:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:44:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:44:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:44:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:44:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:44:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:44:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:44:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:44:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:44:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:44:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:44:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:44:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:44:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:44:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:44:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:45:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:45:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:45:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:45:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:45:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:45:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:45:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:45:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:45:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:45:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:45:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:45:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:45:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:45:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:45:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:45:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:45:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:45:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:45:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:46:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:46:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:46:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:46:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:46:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:46:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:46:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:46:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:46:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:46:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:46:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:46:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:46:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:46:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:46:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:46:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:46:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:46:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:46:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:46:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:46:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:46:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:46:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:46:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:47:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:47:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:47:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:47:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:47:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:47:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:47:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:47:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:47:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:47:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:47:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:47:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:47:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:47:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:47:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:47:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:47:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:47:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:47:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:48:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:48:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:48:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:48:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:48:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:48:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:48:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:48:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:48:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:48:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:48:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:48:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:48:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:48:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:49:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:49:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:49:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:49:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:49:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:49:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:49:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:49:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:49:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:49:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:49:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:49:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:49:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:49:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:50:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:50:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:50:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:50:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:50:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:50:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:50:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:50:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:50:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:50:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:50:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:50:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:50:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:50:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:50:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:50:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:50:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:50:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:50:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:50:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:50:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:50:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:51:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:51:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:51:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:51:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:51:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:51:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:51:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:51:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:51:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:51:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:51:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:51:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:51:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:51:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:51:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:51:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:51:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:51:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:51:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:52:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:52:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:52:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:52:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:52:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:52:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:52:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:52:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:52:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:52:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:52:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:52:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:52:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:52:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:52:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:52:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:52:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:52:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:52:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:52:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:53:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:53:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:53:34 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:53:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:53:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:39 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:53:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:53:44 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:53:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:53:49 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:53:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:53:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:53:54 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:53:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:53:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:53:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:53:59 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:54:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:54:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:54:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:54:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:54:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:54:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:54:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:54:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:54:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:54:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:54:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:54:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:54:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:54:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:54:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:54:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:54:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:54:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:54:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:54:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:54:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:54:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:54:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:54:29 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:55:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:55:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:55:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:55:04 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:55:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:55:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:55:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:55:09 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:55:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:55:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:55:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:55:14 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:55:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt-suraj-cluster-controller-0        Running
Jan 29 12:55:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:55:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:55:19 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-bgrfb        Pending
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-t5d9n        Pending
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-scheduler-7b8fd46d9d-wxqdl        Pending
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-controller-manager-76f447d698-jfj86        Pending
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/pod-checkpointer-4w2rt        Running
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]:         Pod Status:kube-system/kube-apiserver-cglf8        Running
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]: Error: error while checking pod status: timed out waiting for the condition
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]: Tearing down temporary bootstrap control plane...
Jan 29 12:55:24 suraj-cluster-controller-0 bootkube-start[1436]: Error: error while checking pod status: timed out waiting for the condition
Jan 29 12:55:24 suraj-cluster-controller-0 systemd[1]: bootkube.service: Main process exited, code=exited, status=1/FAILURE
Jan 29 12:55:24 suraj-cluster-controller-0 systemd[1]: bootkube.service: Failed with result 'exit-code'.
Jan 29 12:55:24 suraj-cluster-controller-0 systemd[1]: Failed to start Bootstrap a Kubernetes cluster.
```

Some pods are in constant `ContainerCreating` stage:

```console
$ kgp | grep -vi running
NAME                                                READY   STATUS              RESTARTS   AGE
coredns-6f64b7db7-c2j8q                             0/1     ContainerCreating   0          26m
coredns-6f64b7db7-t4l9x                             0/1     ContainerCreating   0          26m
kube-controller-manager-76f447d698-bgrfb            0/1     ContainerCreating   0          26m
kube-controller-manager-76f447d698-jfj86            0/1     ContainerCreating   0          26m
kube-scheduler-7b8fd46d9d-t5d9n                     0/1     ContainerCreating   0          26m
kube-scheduler-7b8fd46d9d-wxqdl                     0/1     ContainerCreating   0          26m
```

And the events tell us what is wrong:

```
$ kge | grep coredns
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-c2j8q                                        no nodes available to schedule pods
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-c2j8q                                        0/1 nodes are available: 1 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-c2j8q                                        0/2 nodes are available: 2 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-c2j8q                                        0/2 nodes are available: 2 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-c2j8q                                        0/4 nodes are available: 4 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-c2j8q                                        0/4 nodes are available: 4 node(s) had taints that the pod didn't tolerate.
<unknown>   Normal    Scheduled                pod/coredns-6f64b7db7-c2j8q                                        Successfully assigned kube-system/coredns-6f64b7db7-c2j8q to suraj-cluster-controller-0
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "bed7e540862cc01e7ca2d7adda01528203cfe5145593ee618391f92967d117b5" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
11m         Normal    SandboxChanged           pod/coredns-6f64b7db7-c2j8q                                        Pod sandbox changed, it will be killed and re-created.
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "90d254c875ec79c95a448c021ce11428963570a2730a13f574c20bdeb9cfcd05" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "b3981a46728733a5821f80b78f4d7c1320278c2887df0e870e92832bc0dfae0d" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "5048b9afc226df82b8ea9a20cea0a13192e9bf7bd59043db2523324b7df683d8" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "d2ebf691c8533ea1a093bc884bf23fb7b3f0030dd3def26178c27aa5097cd98d" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "6b212feecf2b6c036980fc2672740093942e10b1bd8a5acd88f23f1dd1f83755" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
25m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "0592a970be9de354289b9f619e486fb2c9dd18760ae62407a0897df26f6b8549" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
25m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "21ab42add589eb2cb7d33534e0e45168d17e4ecadfff5ba940985887c59955c7" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
25m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "bcff1589709dd2f2d91e393d72cdace3c921b13245b5bc515c5d501583bb7e50" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
88s         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        (combined from similar events): Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "a137d1765dcc48135327f7db106160e1afb6f47233db3b2f81e3fdde923eec46" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-t4l9x                                        no nodes available to schedule pods
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-t4l9x                                        no nodes available to schedule pods
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-t4l9x                                        0/2 nodes are available: 2 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-t4l9x                                        0/2 nodes are available: 2 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-t4l9x                                        0/4 nodes are available: 4 node(s) had taints that the pod didn't tolerate.
<unknown>   Warning   FailedScheduling         pod/coredns-6f64b7db7-t4l9x                                        0/4 nodes are available: 4 node(s) had taints that the pod didn't tolerate.
<unknown>   Normal    Scheduled                pod/coredns-6f64b7db7-t4l9x                                        Successfully assigned kube-system/coredns-6f64b7db7-t4l9x to suraj-cluster-controller-0
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "d4d16f73727545ea03f481add716907cdf274a22e0b312d49f1dd6f5b881a360" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
25m         Normal    SandboxChanged           pod/coredns-6f64b7db7-t4l9x                                        Pod sandbox changed, it will be killed and re-created.
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "879ce71a6656dc6248a99bfbefe6f26bffa9b0ebfd713fb4ab75ba790a362006" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "5f342ea02dc3502851921b18bc7dafac2024a6719d1a0e8fc71b755df37e315d" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "5020f92cb2480fabee04f0ef1e3cda00cd95987847d98ae9b97efe8cdf9738d7" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "fb8a8b3b44176424e5c645968c781c78a9e107d4b9a167890aff9a1161f2b2e0" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "dc5a085f3fd687d6f245b840ce44f21e09bfd374728ecd25e263f148b5c28e23" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
26m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "c56ac9ab26272926531a8d03e52c3f2c7b9f84471ab62a30c2d60b3ecb564e9a" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
25m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "b01c902ed16d70ea3abcb6f3dd116fb85c9e1503745d56a9b85706caf7f2cba6" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
25m         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "942df3b5a3575809c3ad6a840a360e063e3dfe3b0b911a2e799d6bf8a4199605" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
90s         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-t4l9x                                        (combined from similar events): Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "87d9cba787ec643176c31b1d950f6e49af9a83893f7b45a578362afc2db6f3e1" network for pod "coredns-6f64b7db7-t4l9x": networkPlugin cni failed to set up pod "coredns-6f64b7db7-t4l9x_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/
27m         Normal    SuccessfulCreate         replicaset/coredns-6f64b7db7                                       Created pod: coredns-6f64b7db7-c2j8q
27m         Normal    SuccessfulCreate         replicaset/coredns-6f64b7db7                                       Created pod: coredns-6f64b7db7-t4l9x
27m         Normal    ScalingReplicaSet        deployment/coredns                                                 Scaled up replica set coredns-6f64b7db7 to 2
```

Specifically look at this `Warning` event: `88s         Warning   FailedCreatePodSandBox   pod/coredns-6f64b7db7-c2j8q                                        (combined from similar events): Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "a137d1765dcc48135327f7db106160e1afb6f47233db3b2f81e3fdde923eec46" network for pod "coredns-6f64b7db7-c2j8q": networkPlugin cni failed to set up pod "coredns-6f64b7db7-c2j8q_kube-system" network: stat /var/lib/calico/nodename: no such file or directory: check that the calico/node container is running and has mounted /var/lib/calico/`


Same issue with `scheduler` and `controller-manager` pod. And since this crucial piece of the control plane is not up we cannot create anything else.


```console
$ k logs kubelet-d5btt
./hyperkube kubelet --node-ip=10.88.81.5  --anonymous-auth=false --authentication-token-webhook --authorization-mode=Webhook --client-ca-file=/etc/kubernetes/ca.crt --cluster_dns=10.3.0.10 --cluster_domain=cluster.local --cni-conf-dir=/etc/kubernetes/cni/net.d --config=/etc/kubernetes/kubelet.config --kubeconfig=/etc/kubernetes/kubeconfig --lock-file=/var/run/lock/kubelet.lock --network-plugin=cni --pod-manifest-path=/etc/kubernetes/manifests --read-only-port=0 --volume-plugin-dir=/var/lib/kubelet/volumeplugins --node-labels=node.kubernetes.io/node,node.kubernetes.io/storage= --register-with-taints=storage.grafana.com=rook-ceph:NoSchedule

Flag --anonymous-auth has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --authentication-token-webhook has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --authorization-mode has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --cluster-dns has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --cluster-domain has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --read-only-port has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --anonymous-auth has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --authentication-token-webhook has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --authorization-mode has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --cluster-dns has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --cluster-domain has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Flag --read-only-port has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
I0129 12:36:38.472589    2957 server.go:416] Version: v1.17.2
I0129 12:36:38.472812    2957 server.go:494] acquiring file lock on "/var/run/lock/kubelet.lock"
I0129 12:36:38.495004    2957 plugins.go:100] No cloud provider specified.
I0129 12:36:38.530492    2957 server.go:641] --cgroups-per-qos enabled, but --cgroup-root was not specified.  defaulting to /
I0129 12:36:38.531043    2957 container_manager_linux.go:265] container manager verified user specified cgroup-root exists: []
I0129 12:36:38.531057    2957 container_manager_linux.go:270] Creating Container Manager object based on Node Config: {RuntimeCgroupsName: SystemCgroupsName: KubeletCgroupsName: ContainerRuntime:docker CgroupsPerQOS:true CgroupRoot:/ CgroupDriver:cgroupfs KubeletRootDir:/var/lib/kubelet ProtectKernelDefaults:false NodeAllocatableConfig:{KubeReservedCgroupName: SystemReservedCgroupName: ReservedSystemCPUs: EnforceNodeAllocatable:map[pods:{}] KubeReserved:map[] SystemReserved:map[] HardEvictionThresholds:[{Signal:memory.available Operator:LessThan Value:{Quantity:100Mi Percentage:0} GracePeriod:0s MinReclaim:<nil>} {Signal:nodefs.available Operator:LessThan Value:{Quantity:<nil> Percentage:0.1} GracePeriod:0s MinReclaim:<nil>} {Signal:nodefs.inodesFree Operator:LessThan Value:{Quantity:<nil> Percentage:0.05} GracePeriod:0s MinReclaim:<nil>} {Signal:imagefs.available Operator:LessThan Value:{Quantity:<nil> Percentage:0.15} GracePeriod:0s MinReclaim:<nil>}]} QOSReserved:map[] ExperimentalCPUManagerPolicy:none ExperimentalCPUManagerReconcilePeriod:10s ExperimentalPodPidsLimit:-1 EnforceCPULimits:true CPUCFSQuotaPeriod:100ms ExperimentalTopologyManagerPolicy:none}
I0129 12:36:38.531193    2957 fake_topology_manager.go:29] [fake topologymanager] NewFakeManager
I0129 12:36:38.531202    2957 container_manager_linux.go:305] Creating device plugin manager: true
I0129 12:36:38.531227    2957 fake_topology_manager.go:39] [fake topologymanager] AddHintProvider HintProvider:  &{kubelet.sock /var/lib/kubelet/device-plugins/ map[] {0 0} <nil> {{} [0 0 0]} 0x1b1bbe0 0x6e97c50 0x1b1c4b0 map[] map[] map[] map[] map[] 0xc0009874d0 [0 1] 0x6e97c50}
I0129 12:36:38.531262    2957 state_mem.go:36] [cpumanager] initializing new in-memory state store
I0129 12:36:38.531347    2957 state_mem.go:84] [cpumanager] updated default cpuset: ""
I0129 12:36:38.531355    2957 state_mem.go:92] [cpumanager] updated cpuset assignments: "map[]"
I0129 12:36:38.531364    2957 fake_topology_manager.go:39] [fake topologymanager] AddHintProvider HintProvider:  &{{0 0} 0x6e97c50 10000000000 0xc000bcc180 <nil> <nil> <nil> <nil> map[memory:{{104857600 0} {<nil>}  BinarySI}] 0x6e97c50}
I0129 12:36:38.531472    2957 kubelet.go:286] Adding pod path: /etc/kubernetes/manifests
I0129 12:36:38.531517    2957 kubelet.go:311] Watching apiserver
I0129 12:36:38.534075    2957 client.go:75] Connecting to docker on unix:///var/run/docker.sock
I0129 12:36:38.534107    2957 client.go:104] Start docker client with request timeout=2m0s
W0129 12:36:38.535943    2957 docker_service.go:563] Hairpin mode set to "promiscuous-bridge" but kubenet is not enabled, falling back to "hairpin-veth"
I0129 12:36:38.535960    2957 docker_service.go:240] Hairpin mode set to "hairpin-veth"
I0129 12:36:38.577224    2957 docker_service.go:255] Docker cri networking managed by cni
I0129 12:36:38.586299    2957 docker_service.go:260] Docker Info: &{ID:JOHB:5ZIV:PLDM:OXLR:JJRT:BTKW:WJPZ:C3G6:UZIJ:45VV:5QL6:6MC3 Containers:6 ContainersRunning:5 ContainersPaused:0 ContainersStopped:1 Images:3 Driver:overlay2 DriverStatus:[[Backing Filesystem extfs] [Supports d_type true] [Native Overlay Diff true]] SystemStatus:[] Plugins:{Volume:[local] Network:[bridge host macvlan null overlay] Authorization:[] Log:[awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog]} MemoryLimit:true SwapLimit:true KernelMemory:true KernelMemoryTCP:false CPUCfsPeriod:true CPUCfsQuota:true CPUShares:true CPUSet:true PidsLimit:false IPv4Forwarding:true BridgeNfIptables:true BridgeNfIP6tables:true Debug:false NFd:49 OomKillDisable:true NGoroutines:64 SystemTime:2020-01-29T12:36:38.578510606Z LoggingDriver:json-file CgroupDriver:cgroupfs NEventsListener:0 KernelVersion:4.19.84-flatcar OperatingSystem:Flatcar Container Linux by Kinvolk 2247.7.0 (Rhyolite) OSType:linux Architecture:x86_64 IndexServerAddress:https://index.docker.io/v1/ RegistryConfig:0xc0007380e0 NCPU:32 MemTotal:135146491904 GenericResources:[] DockerRootDir:/var/lib/docker HTTPProxy: HTTPSProxy: NoProxy: Name:suraj-cluster-helium-worker-1 Labels:[] ExperimentalBuild:false ServerVersion:18.06.3-ce ClusterStore: ClusterAdvertise: Runtimes:map[runc:{Path:docker-runc Args:[]}] DefaultRuntime:runc Swarm:{NodeID: NodeAddr: LocalNodeState:inactive ControlAvailable:false Error: RemoteManagers:[] Nodes:0 Managers:0 Cluster:<nil> Warnings:[]} LiveRestoreEnabled:false Isolation: InitBinary:docker-init ContainerdCommit:{ID:468a545b9edcd5932818eb9de8e72413e616e86e Expected:468a545b9edcd5932818eb9de8e72413e616e86e} RuncCommit:{ID:a592beb5bc4c4092b1b1bac971afed27687340c5 Expected:a592beb5bc4c4092b1b1bac971afed27687340c5} InitCommit:{ID:fec3683b971d9c3ef73f284f176672c44b448662 Expected:fec3683b971d9c3ef73f284f176672c44b448662} SecurityOptions:[name=seccomp,profile=default name=selinux] ProductLicense: Warnings:[]}
I0129 12:36:38.586415    2957 docker_service.go:273] Setting cgroupDriver to cgroupfs
I0129 12:36:38.597473    2957 remote_runtime.go:59] parsed scheme: ""
I0129 12:36:38.597489    2957 remote_runtime.go:59] scheme "" not registered, fallback to default scheme
I0129 12:36:38.597538    2957 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{/var/run/dockershim.sock 0  <nil>}] <nil>}
I0129 12:36:38.597549    2957 clientconn.go:577] ClientConn switching balancer to "pick_first"
I0129 12:36:38.597590    2957 remote_image.go:50] parsed scheme: ""
I0129 12:36:38.597598    2957 remote_image.go:50] scheme "" not registered, fallback to default scheme
I0129 12:36:38.597622    2957 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{/var/run/dockershim.sock 0  <nil>}] <nil>}
I0129 12:36:38.597628    2957 clientconn.go:577] ClientConn switching balancer to "pick_first"
E0129 12:36:58.908317    2957 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
I0129 12:36:58.909975    2957 kuberuntime_manager.go:211] Container runtime docker initialized, version: 18.06.3-ce, apiVersion: 1.38.0
I0129 12:36:58.911288    2957 server.go:1113] Started kubelet
E0129 12:36:58.911390    2957 kubelet.go:1302] Image garbage collection failed once. Stats initialization may not have completed yet: failed to get imageFs info: unable to find data in memory cache
I0129 12:36:58.911390    2957 server.go:143] Starting to listen on 0.0.0.0:10250
I0129 12:36:58.912990    2957 server.go:354] Adding debug handlers to kubelet server.
I0129 12:36:58.914267    2957 fs_resource_analyzer.go:64] Starting FS ResourceAnalyzer
I0129 12:36:58.914349    2957 volume_manager.go:265] Starting Kubelet Volume Manager
I0129 12:36:58.914420    2957 desired_state_of_world_populator.go:138] Desired state populator starts to run
I0129 12:36:58.934873    2957 status_manager.go:157] Starting to sync pod status with apiserver
I0129 12:36:58.934911    2957 kubelet.go:1820] Starting kubelet main sync loop.
E0129 12:36:58.934994    2957 kubelet.go:1844] skipping pod synchronization - [container runtime status check may not have completed yet, PLEG is not healthy: pleg has yet to be successful]
I0129 12:36:59.014457    2957 kubelet_node_status.go:294] Setting node annotation to enable volume controller attach/detach
I0129 12:36:59.014526    2957 kuberuntime_manager.go:981] updating runtime config through cri with podcidr 10.2.2.0/24
I0129 12:36:59.015035    2957 docker_service.go:355] docker cri received runtime config &RuntimeConfig{NetworkConfig:&NetworkConfig{PodCidr:10.2.2.0/24,},}
I0129 12:36:59.015295    2957 kubelet_network.go:77] Setting Pod CIDR:  -> 10.2.2.0/24
I0129 12:36:59.016986    2957 kubelet_node_status.go:70] Attempting to register node suraj-cluster-helium-worker-1
I0129 12:36:59.030456    2957 kubelet_node_status.go:112] Node suraj-cluster-helium-worker-1 was previously registered
I0129 12:36:59.030718    2957 kubelet_node_status.go:73] Successfully registered node suraj-cluster-helium-worker-1
E0129 12:36:59.035143    2957 kubelet.go:1844] skipping pod synchronization - container runtime status check may not have completed yet
I0129 12:36:59.035527    2957 setters.go:535] Node became not ready: {Type:Ready Status:False LastHeartbeatTime:2020-01-29 12:36:59.035506577 +0000 UTC m=+20.617487980 LastTransitionTime:2020-01-29 12:36:59.035506577 +0000 UTC m=+20.617487980 Reason:KubeletNotReady Message:container runtime status check may not have completed yet}
I0129 12:36:59.036809    2957 cpu_manager.go:173] [cpumanager] starting with none policy
I0129 12:36:59.036822    2957 cpu_manager.go:174] [cpumanager] reconciling every 10s
I0129 12:36:59.036853    2957 policy_none.go:43] [cpumanager] none policy: Start
I0129 12:36:59.038716    2957 plugin_manager.go:114] Starting Kubelet Plugin Manager
I0129 12:36:59.317502    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "var-lib-kubelet" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-var-lib-kubelet") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.317549    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "ssl-certs-host" (UniqueName: "kubernetes.io/host-path/76eaf18b-abd1-4d56-a520-abc7ca56c730-ssl-certs-host") pod "kube-proxy-bnspv" (UID: "76eaf18b-abd1-4d56-a520-abc7ca56c730") 
I0129 12:36:59.317607    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "kube-proxy-token-t8qpn" (UniqueName: "kubernetes.io/secret/76eaf18b-abd1-4d56-a520-abc7ca56c730-kube-proxy-token-t8qpn") pod "kube-proxy-bnspv" (UID: "76eaf18b-abd1-4d56-a520-abc7ca56c730") 
I0129 12:36:59.317650    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "xtables-lock" (UniqueName: "kubernetes.io/host-path/c134dab3-721a-454f-9a34-0170cee34d6a-xtables-lock") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.317717    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "cni-bin-dir" (UniqueName: "kubernetes.io/host-path/c134dab3-721a-454f-9a34-0170cee34d6a-cni-bin-dir") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.317808    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "calico-node-token-w8x8s" (UniqueName: "kubernetes.io/secret/c134dab3-721a-454f-9a34-0170cee34d6a-calico-node-token-w8x8s") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.317834    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "logs" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-logs") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.317856    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "default-token-fhjbn" (UniqueName: "kubernetes.io/secret/3e05378e-034c-4b35-9d43-ca3a553c9861-default-token-fhjbn") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.317877    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "var-run-calico" (UniqueName: "kubernetes.io/host-path/c134dab3-721a-454f-9a34-0170cee34d6a-var-run-calico") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.317934    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "cni-conf-dir" (UniqueName: "kubernetes.io/host-path/c134dab3-721a-454f-9a34-0170cee34d6a-cni-conf-dir") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.317994    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "coreos-var-lib-cni" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-coreos-var-lib-cni") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318041    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "coreos-opt-cni-bin" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-coreos-opt-cni-bin") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318089    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "sys" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-sys") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318126    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "lib-modules" (UniqueName: "kubernetes.io/host-path/76eaf18b-abd1-4d56-a520-abc7ca56c730-lib-modules") pod "kube-proxy-bnspv" (UID: "76eaf18b-abd1-4d56-a520-abc7ca56c730") 
I0129 12:36:59.318168    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "dev" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-dev") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318211    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "run" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-run") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318251    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "var-lib-docker" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-var-lib-docker") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318270    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "os-release" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-os-release") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318334    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "kubeconfig" (UniqueName: "kubernetes.io/configmap/76eaf18b-abd1-4d56-a520-abc7ca56c730-kubeconfig") pod "kube-proxy-bnspv" (UID: "76eaf18b-abd1-4d56-a520-abc7ca56c730") 
I0129 12:36:59.318365    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "lib-modules" (UniqueName: "kubernetes.io/host-path/c134dab3-721a-454f-9a34-0170cee34d6a-lib-modules") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.318386    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "var-lib-calico" (UniqueName: "kubernetes.io/host-path/c134dab3-721a-454f-9a34-0170cee34d6a-var-lib-calico") pod "calico-node-2gwhd" (UID: "c134dab3-721a-454f-9a34-0170cee34d6a") 
I0129 12:36:59.318422    2957 reconciler.go:209] operationExecutor.VerifyControllerAttachedVolume started for volume "etc-kubernetes" (UniqueName: "kubernetes.io/host-path/3e05378e-034c-4b35-9d43-ca3a553c9861-etc-kubernetes") pod "kubelet-d5btt" (UID: "3e05378e-034c-4b35-9d43-ca3a553c9861") 
I0129 12:36:59.318445    2957 reconciler.go:156] Reconciler: start to sync state
....
Removed all the "connection refused"
....
E0129 12:56:31.370771    2957 remote_runtime.go:351] ExecSync 0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450 '/bin/calico-node -felix-ready --bird-ready' from runtime service failed: rpc error: code = Unknown desc = container not running (0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450)
E0129 12:56:31.371870    2957 remote_runtime.go:351] ExecSync 0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450 '/bin/calico-node -felix-ready --bird-ready' from runtime service failed: rpc error: code = Unknown desc = container not running (0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450)
E0129 12:56:31.372770    2957 remote_runtime.go:351] ExecSync 0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450 '/bin/calico-node -felix-ready --bird-ready' from runtime service failed: rpc error: code = Unknown desc = container not running (0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450)
W0129 12:56:31.372811    2957 prober.go:108] No ref for container "docker://0326c4e2c14739ce2be48d1ec7dbee3a79b6c1a494780878364a82556e43f450" (calico-node-2gwhd_kube-system(c134dab3-721a-454f-9a34-0170cee34d6a):calico-node)
E0129 12:56:39.714836    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:56:49.720099    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:56:59.725105    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:57:09.730460    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:57:19.736284    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:57:29.133776    2957 event.go:217] Unable to write event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"calico-node-2gwhd.15ee5cc9d53eb791", GenerateName:"", Namespace:"kube-system", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Pod", Namespace:"kube-system", Name:"calico-node-2gwhd", UID:"c134dab3-721a-454f-9a34-0170cee34d6a", APIVersion:"v1", ResourceVersion:"834", FieldPath:"spec.containers{calico-node}"}, Reason:"Unhealthy", Message:"Liveness probe failed: calico/node is not ready: felix is not ready: readiness probe reporting 503", Source:v1.EventSource{Component:"kubelet", Host:"suraj-cluster-helium-worker-1"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbf847e134b4af591, ext:1142771443304, loc:(*time.Location)(0x6e79d80)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbf847e134b4af591, ext:1142771443304, loc:(*time.Location)(0x6e79d80)}}, Count:1, Type:"Warning", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}' (retry limit exceeded!)
' (may retry after sleeping)
E0129 12:57:29.741698    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:57:39.747217    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
' (may retry after sleeping)
E0129 12:57:49.752391    2957 kubelet_node_status.go:389] Unable to update node status: update node status exceeds retry count
E0129 12:58:09.390632    2957 reflector.go:307] object-"kube-system"/"calico-config": Failed to watch *v1.ConfigMap: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/namespaces/kube-system/configmaps?allowWatchBookmarks=true&fieldSelector=metadata.name%3Dcalico-config&resourceVersion=253&timeout=9m40s&timeoutSeconds=580&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.391439    2957 reflector.go:307] object-"kube-system"/"kube-proxy-token-t8qpn": Failed to watch *v1.Secret: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/namespaces/kube-system/secrets?allowWatchBookmarks=true&fieldSelector=metadata.name%3Dkube-proxy-token-t8qpn&resourceVersion=396&timeout=7m28s&timeoutSeconds=448&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.392628    2957 reflector.go:307] object-"kube-system"/"default-token-fhjbn": Failed to watch *v1.Secret: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/namespaces/kube-system/secrets?allowWatchBookmarks=true&fieldSelector=metadata.name%3Ddefault-token-fhjbn&resourceVersion=396&timeout=9m9s&timeoutSeconds=549&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.393554    2957 reflector.go:307] k8s.io/kubernetes/pkg/kubelet/kubelet.go:458: Failed to watch *v1.Node: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/nodes?allowWatchBookmarks=true&fieldSelector=metadata.name%3Dsuraj-cluster-helium-worker-1&resourceVersion=3595&timeoutSeconds=378&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.394740    2957 reflector.go:307] k8s.io/client-go/informers/factory.go:135: Failed to watch *v1beta1.RuntimeClass: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/apis/node.k8s.io/v1beta1/runtimeclasses?allowWatchBookmarks=true&resourceVersion=1&timeout=5m27s&timeoutSeconds=327&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.395853    2957 reflector.go:307] k8s.io/kubernetes/pkg/kubelet/kubelet.go:449: Failed to watch *v1.Service: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/services?allowWatchBookmarks=true&resourceVersion=295&timeoutSeconds=519&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.396921    2957 reflector.go:307] k8s.io/client-go/informers/factory.go:135: Failed to watch *v1beta1.CSIDriver: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/apis/storage.k8s.io/v1beta1/csidrivers?allowWatchBookmarks=true&resourceVersion=1&timeout=7m3s&timeoutSeconds=423&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.398065    2957 reflector.go:307] object-"kube-system"/"calico-node-token-w8x8s": Failed to watch *v1.Secret: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/namespaces/kube-system/secrets?allowWatchBookmarks=true&fieldSelector=metadata.name%3Dcalico-node-token-w8x8s&resourceVersion=396&timeout=8m29s&timeoutSeconds=509&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.399149    2957 reflector.go:307] k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:46: Failed to watch *v1.Pod: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/pods?allowWatchBookmarks=true&fieldSelector=spec.nodeName%3Dsuraj-cluster-helium-worker-1&resourceVersion=2873&timeoutSeconds=461&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.400164    2957 reflector.go:307] object-"kube-system"/"kubeconfig-in-cluster": Failed to watch *v1.ConfigMap: Get https://suraj-cluster-private.dev.lokomotive-k8s.net.:6443/api/v1/namespaces/kube-system/configmaps?allowWatchBookmarks=true&fieldSelector=metadata.name%3Dkubeconfig-in-cluster&resourceVersion=253&timeout=6m26s&timeoutSeconds=386&watch=true: net/http: TLS handshake timeout
E0129 12:58:09.627120    2957 controller.go:135] failed to ensure node lease exists, will retry in 7s, error: leases.coordination.k8s.io "suraj-cluster-helium-worker-1" is forbidden: User "kubelet" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "kube-node-lease"
W0129 12:58:09.627121    2957 status_manager.go:530] Failed to get status for pod "calico-node-2gwhd_kube-system(c134dab3-721a-454f-9a34-0170cee34d6a)": pods "calico-node-2gwhd" is forbidden: User "kubelet" cannot get resource "pods" in API group "" in the namespace "kube-system"
E0129 12:58:09.627279    2957 kubelet_node_status.go:402] Error updating node status, will retry: error getting node "suraj-cluster-helium-worker-1": nodes "suraj-cluster-helium-worker-1" is forbidden: User "kubelet" cannot get resource "nodes" in API group "" at the cluster scope
E0129 12:58:09.627232    2957 event.go:263] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"calico-node-2gwhd.15ee5cc9e4f8015f", GenerateName:"", Namespace:"kube-system", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Pod", Namespace:"kube-system", Name:"calico-node-2gwhd", UID:"c134dab3-721a-454f-9a34-0170cee34d6a", APIVersion:"v1", ResourceVersion:"834", FieldPath:"spec.containers{calico-node}"}, Reason:"Unhealthy", Message:"Readiness probe failed: calico/node is not ready: felix is not ready: readiness probe reporting 503", Source:v1.EventSource{Component:"kubelet", Host:"suraj-cluster-helium-worker-1"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbf847e135b043f5f, ext:1143035244599, loc:(*time.Location)(0x6e79d80)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbf847e135b043f5f, ext:1143035244599, loc:(*time.Location)(0x6e79d80)}}, Count:1, Type:"Warning", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events is forbidden: User "kubelet" cannot create resource "events" in API group "" in the namespace "kube-system"' (will not retry!)
E0129 12:58:09.639440    2957 event.go:263] Server rejected event '&v1.Event{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"calico-node-2gwhd.15ee5cc9d53eb791", GenerateName:"", Namespace:"kube-system", SelfLink:"", UID:"", ResourceVersion:"", Generation:0, CreationTimestamp:v1.Time{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, DeletionTimestamp:(*v1.Time)(nil), DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ClusterName:"", ManagedFields:[]v1.ManagedFieldsEntry(nil)}, InvolvedObject:v1.ObjectReference{Kind:"Pod", Namespace:"kube-system", Name:"calico-node-2gwhd", UID:"c134dab3-721a-454f-9a34-0170cee34d6a", APIVersion:"v1", ResourceVersion:"834", FieldPath:"spec.containers{calico-node}"}, Reason:"Unhealthy", Message:"Liveness probe failed: calico/node is not ready: felix is not ready: readiness probe reporting 503", Source:v1.EventSource{Component:"kubelet", Host:"suraj-cluster-helium-worker-1"}, FirstTimestamp:v1.Time{Time:time.Time{wall:0xbf847e134b4af591, ext:1142771443304, loc:(*time.Location)(0x6e79d80)}}, LastTimestamp:v1.Time{Time:time.Time{wall:0xbf847e15cb575c4f, ext:1152772256059, loc:(*time.Location)(0x6e79d80)}}, Count:2, Type:"Warning", EventTime:v1.MicroTime{Time:time.Time{wall:0x0, ext:0, loc:(*time.Location)(nil)}}, Series:(*v1.EventSeries)(nil), Action:"", Related:(*v1.ObjectReference)(nil), ReportingController:"", ReportingInstance:""}': 'events "calico-node-2gwhd.15ee5cc9d53eb791" is forbidden: User "kubelet" cannot patch resource "events" in API group "" in the namespace "kube-system"' (will not retry!)
E0129 12:58:09.639643    2957 kubelet_node_status.go:402] Error updating node status, will retry: error getting node "suraj-cluster-helium-worker-1": nodes "suraj-cluster-helium-worker-1" is forbidden: User "kubelet" cannot get resource "nodes" in API group "" at the cluster scope
I0129 13:10:01.542801    2957 log.go:172] http: superfluous response.WriteHeader call from k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/httplog.(*respLogger).WriteHeader (httplog.go:197)
```

My conjecture is that once the self hosted kubelet came up the pods could not start, at all.

Reply:
Update: I deleted the daemonset kubelet (and pods manually because the deletion didn't cascading deletion of the pods) then things went back to normal, because bootstrap kubelet has taken over.

Reply:
Ah yes, I forgot about that. Well, you could set `"nodename_file_optional": true,` in Calico configuration, then it will be fine, but I don't know what are full implications of that, so I guess it's fine to leave it for now. Does it also work when Calico directory is not present on the host? So when using `flannel` for networking?

Reply:
I am not quite sure, if we need such a setting for lokomotive. 

But I am assuming that it should work, because this dir is specific to calico.

Reply:
I meant, what happens when the directory is not present on the host? Does the pod starts just fine then?

Reply:
Are you speaking about the kubelet pod or the app pod that is failing?

Reply:
kubelet pod.

Reply:
So I tried it again, deleted the ds & pods, deleted the dir and recreated the ds again. So I see dir created again but I don't see any file in it which previously existed.